docs/examples/docker-compose.override.yml

```diff
@@ -0,0 +1,4 @@
+version: "3.7"
```

Comment:
Does this work given the main compose is specifying a local volume?

I though you would have to have this in the main compose file and specifying keys in the dict to say it's a local volume. Then override it here in the .override with no args to use /var/lib/docker volume

Reply:
It works yes.  If you specify a path in the env var it's a filesystem volume, if you specify a flat name (no slashes) it looks for a named volume, in which case you would need to name it in the volumes: list.  As I'm considering that optional at the moment, I've put the snippet for volumes in override rather than in core as I don't want to make anything like that required at this stage, remembering it's easier to add things in an override than to take them away if they're in the core yml.